src/tools/fuzzing.h

```diff
@@ -994,39 +978,34 @@ class TranslateToFuzzReader {
   }
 
   Expression* _makeunreachable() {
-    switch (upTo(15)) {
-      case 0:
-        return makeBlock(Type::unreachable);
-      case 1:
-        return makeIf(Type::unreachable);
-      case 2:
-        return makeLoop(Type::unreachable);
-      case 3:
-        return makeBreak(Type::unreachable);
-      case 4:
-        return makeCall(Type::unreachable);
-      case 5:
-        return makeCallIndirect(Type::unreachable);
-      case 6:
-        return makeLocalSet(Type::unreachable);
-      case 7:
-        return makeStore(Type::unreachable);
-      case 8:
-        return makeUnary(Type::unreachable);
-      case 9:
-        return makeBinary(Type::unreachable);
-      case 10:
-        return makeSelect(Type::unreachable);
-      case 11:
-        return makeSwitch(Type::unreachable);
-      case 12:
-        return makeDrop(Type::unreachable);
-      case 13:
-        return makeReturn(Type::unreachable);
-      case 14:
-        return makeUnreachable(Type::unreachable);
-    }
-    WASM_UNREACHABLE("unexpected value");
+    using Self = TranslateToFuzzReader;
+    auto options =
+      FeatureOptions<Expression* (Self::*)(Type)>().add(FeatureSet::MVP,
+                                                        &Self::makeBlock,
+                                                        LastIsImportant,
```

Comment:
Would it be possible to do this with `std::initializer_list` or similar so we write something like

```
FeatureOptions<...>()
    .add(FeatureSet::MVP, 
         {item, 2},
         normal_item,
         {foo, 4},
         ...
```

I think that would be more readable than interspersing the numbers among the items.

Reply:
I couldn't get it to compile that way no matter how hard I tried, sadly. But with a named struct I could do it, let me know what you think. (Is there a way to make that work?)